PowerShellTools/PowerShellToolsPackage.cs

```diff
@@ -185,6 +189,7 @@ private void InitializeInternal()
             }
 
             InitializePowerShellHost();
+            EstablishProcessConnection();

```

Comment:
We'd better to establish connection ahead of initializing powershellhost in which some operation(around debugger) needs to happen on remote process.